### PR TITLE
window: Rework widgetry

### PR DESCRIPTION
**Pull request by BrainBlasted**
- **window: Rework widget hierarchy**
- **window: Make searchbar a ToolbarView top bar**
- **window: Use bidirectional property binding for search**
- **window: Simplify and sync sensitivity**
- **window: Add status pages for search/no voices**

### CODE REVIEW

File: src/window.py [hunk 1]
```diff
@@ -71,6 +74,8 @@ def __init__(self, **kwargs):
         self.vstore.connect("populated", self._on_vstore_populated)
         self.vstore.populate()
 
+        self.instructions_button.connect("clicked", self._on_instructions_button_clicked)
```
Comment (eeejay) [inline src/window.py]: format with `black`

**Comment by eeejay** [comment]
This PR is amazing! Thank you for working on this. Can you please format with `black` and push?

**Comment by BrainBlasted** [comment]
Done :)

**Comment by eeejay** [comment]
than you!

**Pull request merged**